opentelemetry_collector/receiver/dockerstats/scraper.go

```diff
@@ -27,6 +27,20 @@ var (
 		Description: "Name of the container (or ID if name is not available)",
 	}
 
+	cpuUsageDesc = &mpb.MetricDescriptor{
+		Name:        "container/cpu/usage",
+		Description: "Total CPU time consumed",
+		Unit:        "ns",
```

Comment:
This should be nanoseconds according to the list of well known streamz units at monitoring/streamz/public/metadata.h. Some of the other metric definitions have the same problem. About your reply to my comment on unit names in the previous pull request, I think the google naming conventions should take precedent over opencensus's.

Reply:
Disagree with that assessment, since these metrics are defined using OpenCensus, not streamz.

Don't care enough to insist on it though.